advisories/github-reviewed/2021/08/GHSA-mmc9-pwm7-qj5w/GHSA-mmc9-pwm7-qj5w.json

```diff
@@ -20,12 +20,6 @@
         "ecosystem": "crates.io",
         "name": "rand_core"
       },
-      "ecosystem_specific": {
-        "affected_functions": [
-          "rand_core::BlockRng::fill_bytes",
-          "rand_core::BlockRng::next_u64"
-        ]
-      },
```

Comment:
Why is this removed?

Reply:
I did not intend to remove that, sorry. I used the improvement suggestion form, not a real repo checkout.

Reply:
In fact, I cannot figure out how to edit the PR at all. Can you remove the chunk here and below?